### DISCUSSION TITLE
rapports : format des durées

### DISCUSSION THREAD

**Discussion by shermelin** [JeromeDevome/GRR#390] [Idées]
Bonjour,

Nous avons récemment migré nos outils de réservation vieillissant vers GRR et en sommes bien contents. Merci pour ce travail.

J'aimerais soulever un légère difficulté : lors de l'émission des rapports, les durées sont écrites sous un format très "humain" (x jours, y heures). Serait-il possible de choisir d'exporter dans un format plus "simple" (tout en minutes par exemple) ?
Je me suis plongé dans le code pour essayer de proposer un patch, mais me suis un peu perdu sur la fin (et craignais des effets de bord...). Je suis partant pour remettre les mains dans le code avec quelques indices !
Merci

**Comment by ynaessens** [comment]
Bonjour,
j'imagine que ce serait surtout utile pour l'export en csv.
À mon avis il suffit de surcharger la fonction describe_span() en ajoutant un paramètre testant si c'est un export csv ou html et calculant en minutes ou en éditant la description comme actuellement.
Je vous laisse réfléchir à cette proposition et proposer une solution si vous êtes prêt sur la question.
Cordialement,
YN

**Comment by JeromeDevome** [comment]
Bonjour,
Je pense qu'il faut laisser le choix à l'utilisateur dans le formulaire.
Attention aux créneaux de réservation basés sur des intitulés.
